asn1ate/pyasn1gen.py

```diff
@@ -283,7 +287,10 @@ def inline_simple_type(self, t):
         return type_expr
 
     def inline_defined_type(self, t):
-        return _translate_type(t.type_name) + '()'
+        if t.module_name is None or t.module_name == self.sema_module.name:
+            return _translate_type(t.type_name) + '()'
```

Comment:
Since you do the `_translate_type` call in both branches, can you do it first, something like:

    translated_type =  _translate_type(t.type_name) + '()'
    if t.module_name and t.module_name != self.sema_module.name:
        translated_type = _sanitize_module(t.module_name) + '.' + translated_type
    return translated_type

? Thanks.